src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -160,7 +162,7 @@ export const TimelineNav = ({
       prevActiveIndex !== activeIndex
     ) {
       setActiveIndexState(activeIndex);
-      timelineNavItemRefs[activeIndex].current.focus();
+      timelineNavItemRefs[activeIndex].current?.focus();
```

Comment:
Now that we have better types, the ref isn't `any` anymore, and we actually have to handle the `null` case.